lib/supported-features.js

```diff
@@ -5,6 +5,8 @@ const advertisedStyles = [
   'flat',
   'flat-square',
   'for-the-badge',
+  'flat-logo',
+  'flat-square-logo',
```

Comment:
Do you think `popout` and `popout-square` might be clearer names for these formats?

Reply:
Yeah good idea, i couldn't think of what to call them, but `popout` is a great choice.